### PR TITLE
Rwc2019

### PR DESCRIPTION
**Pull request by MatthijsBurgh**

### CODE REVIEW

File: netpbm/install.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+- type: system
+  name: libnetpbm10-dev
```
Comment (LoyVanBeek) [inline netpbm/install.yaml]: What is netpbm for?

File: tue-common/rsettings_file [hunk 2]
```diff
@@ -2,12 +2,13 @@
 
 # Robot environment
 #export ROBOT_ENV=reo2016
-export ROBOT_ENV=robotics_testlabs
+#export ROBOT_ENV=robotics_testlabs
```
Comment (PetervDooren) [inline tue-common/rsettings_file]: Default should be set to robotics_testlabs

File: netpbm/install.yaml (hunk 1 above)
Comment (MatthijsBurgh) [reply netpbm/install.yaml]: It is used by the some navigation packages. We don't use these packages, but they are in the fork of the nav stack.

Comment (LoyVanBeek) [reply netpbm/install.yaml]: OK

File: tue-common/rsettings_file (hunk 2 above)
Comment (LoyVanBeek) [reply tue-common/rsettings_file]: +1

**Pull request merged**